Dockerfile.ci

```diff
@@ -248,6 +248,7 @@ RUN mkdir -p ~/.local/bin \
        "  for ruby_version in \$(rbenv whence ruby); do\n" \
        "    echo \"Switching to \${ruby_version}\"\n" \
        "    rbenv shell \"\${ruby_version}\"\n" \
+       "    gem update --system\n" \
```

Comment:
Have you needed this line for it to work? I remember having had issues with some broken gem updates in the past, so I went with the gem version shipping which each ruby version.

Reply:
Yes, this is needed for ruby 2.7. With the version installed by default, it crashes.